### ISSUE TITLE
Replace Mailcatcher with MailDev

### ISSUE THREAD

**Issue by markshust** [markshust/docker-magento#604]
More research is needed, but this looks like a better local mail delivery solution: http://maildev.github.io/maildev/

**Comment by soulteary** [comment]
I agree with you, we can build maildev together if you want. @markshust

**Comment by markshust** [comment]
@soulteary I'd be open to taking in a PR for this!
